rvepak/mail_room_object.py

```diff
@@ -0,0 +1,182 @@
+import os
+import sys
+import datetime
+import prettytable
+
+donors_history_file = 'donors_history.txt'
+allowed_response_list = ['Send a Thank You', 'Create a Report', 'quit']
+
+
+def pretty_report(header, nestedList):
+    tbl = prettytable.PrettyTable(header)
+    tbl.padding_width = 1
+    for lst in nestedList:
+        tbl.add_row(lst)
+    return tbl
+
+
+class File(object):
+    def __init__(self, in_file):
+        self.in_file = in_file
+
+    def read_file_object(self):
+        return open(self.in_file, 'r')
+
+    def write_file_object(self):
+        return open(self.in_file, 'w')
+
+    def append_file_object(self):
+        return open(self.in_file, 'a')
+
+    def read_all_lines(self, read_object):
+        return read_object.readlines()
+
+    def close_file_object(self, in_object):
+        return in_object.close()
```

Comment:
if you specify a `def __enter__():` and `def __exit__():` functions you can actually use this as a context manager using `with` i believe.